.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,86 +0,0 @@
-# Description
```

Comment:
why's this file being deleted

Reply:
Changes were made to master branch. Could you checkout your feature branch from staging instead and then make a new PR